winpr/libwinpr/CMakeLists.txt

```diff
@@ -133,6 +133,8 @@ endif()
 
 add_definitions(${WINPR_DEFINITIONS})
 if (WIN32)
+        # With some versions of msvcrt, you need msvcr100 to get wmemmove_s
+        list(APPEND WINPR_LIBS "msvcr100")
```

Comment:
How can this work if the runtime is not the one linked to? (See pull request tester failure)

Reply:
> How can this work if the runtime is not the one linked to? (See pull request tester failure)

The Jenkins log is gone, will check again after new pull.